src/config.ts

```diff
@@ -406,6 +406,16 @@ export const RFQ_PROXY_PORT: number | undefined = _.isEmpty(process.env.RFQ_PROX
     ? undefined
     : assertEnvVarType('RFQ_PROXY_PORT', process.env.RFQ_PROXY_PORT, EnvVarType.Port);
 
+export const ORIGIN_REGISTRY_ADDRESS: string = _.isEmpty(process.env.ORIGIN_REGISTRY_ADDRESS)
```

Comment:
I think @phil-ociraptor already created this environment variable right?

Reply:
correct - it's called `META_TX_WORKER_REGISTRY`